spec/sql_translator_spec.rb

```diff
@@ -21,25 +21,4 @@
     recipe_to_sql = Polo::SqlTranslator.new(recipe).to_sql.first
     expect(recipe_to_sql).to include(%q{'{"quality":"ok"}'}) # JSON, not YAML
```

Comment:
it would probably make sense to write some simple tests here just to guarantee that we're sending calls our strategy object. Nothing too fancy, but just to keep things consistent.